### DISCUSSION TITLE
Confused by semantic token categories?

### DISCUSSION THREAD

**Discussion by florianmartens** [chakra-ui/chakra-ui#5870] [Q&A]
I'm a little confused by the docs on Semantic Tokens. Lets say we define semantic tokens like this:

```
const theme = extendTheme({
  semanticTokens: {
    colors: {
      text: {
        default: "green.500",
        _dark: "blue.500"
      },
      card: {
        default: "gray.50",
        _dark: "gray.700"
      }
    },
    shadows: {
      card: {
        default: "md",
        _dark: "none"
      }
    },
    radii: {
      card: "lg"
    }
  }
});
```

In the example, we're using the categories/top level keys `colors`, `shadows`and `radii`. These seem to be the the top level keys from the `defaultTheme` object. When using the semanticTokens, we do this:

```
<Box shadow="card" p={4} rounded="card" bg="card">
        <Text color="text">Semantic Tokens are Awesome!</Text>
 </Box>
```

This works for colors, but what happens if I want to specify a semantic token for e.g. `padding="distance-sm"`. What category would I use for this? Do I need the categories (`colors`, `shadows`...) or can I simply omit them? As is, I don't think the docs are very clear on this.

Doing this does not work:

```
const theme = extendTheme({
  semanticTokens: {
    distance_sm: 8,
});
```

```
<Box p="distance_sm" />
```

**Comment by arnoldsandoval** [comment]
@florianmartens in your case I believe you would define `distance_sm` within the `semanticTokens.space` ([See theme.types.ts](https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/theme.types.ts#L116)) key.

I do have a similar question for things that cannot be found within the `Foundations`, for example, if I wanted to assign a gradient as a semantic token, how would I do that?

**Reply by florianmartens** [reply]
hey @arnoldsandoval , great point! Maybe someone will have the answer.

**Reply by arnoldsandoval** [reply]
@bhardy suggested I use [layerStyles](https://chakra-ui.com/docs/styled-system/features/text-and-layer-styles) in the case of gradients. I'm wondering if gradients were deliberately omitted from the first pass of semantic tokens, or is this intentional? Should we stick with layer styles in this case?

**Comment by imabp** [comment]
Yes, I am also looking for semantic token for space. 
May be @segunadebayo have the answer?

**Reply by florianmartens** [reply]
I think @arnoldsandoval provided the correct answer for `space` tokens.

**Reply by imabp** [reply]
thank you, it works :)
